airflow/operators/s3_to_sql.py

```diff
@@ -0,0 +1,117 @@
+#
```

Comment:
In my opinion this file should be in the Amazon provider package. It should be in `/airflow/airflow/providers/amazon/aws/transfers`